setup.cfg

```diff
@@ -41,36 +41,25 @@ docker-compose-v1 =
     docker-compose >=1.27.3, <2.0
 tests =
     requests >=2.22.0, <3.0
-    pytest-pylint >=0.14.1, <1.0
+    mypy >=0.500, <2.000
     pytest-pycodestyle >=2.0.0, <3.0
+    pytest-mypy >=0.10, <1.0
+    types-requests >=2.31, <3.0
+    types-setuptools >=69.0, <70.0
 
 [options.entry_points]
 pytest11 =
     docker = pytest_docker
 
 [tool:pytest]
-addopts = --verbose --pylint-rcfile=setup.cfg
-# --pylint --pycodestyle
+addopts = --verbose --mypy --pycodestyle
```

Comment:
I'd expect the `--mypy` flag no to be enabled by default, just as `--pycodestyle` or `--pylint`

Reply:
I initially removed pylint since the usage of it was commented out and it really was broken when uncommented and run on master branch. Same for pycodestyle. With pycodestyle it was a 2min easy fix so I kept it. With pylint the issue seemed harder to fix and since it could not be used I removed the dependency. But it got me thinking that it can't be that hard and that I can't be that lazy so I added it back and fixed the issues with it. 

Let me disagree in the most possible way please, that it should not be enabled by default. I kind of strongly think the opposite. The pylint, pycodestyle and now mypy are included to enforce some coding standards. If you want to test the code without it, feel free to disable it, but for the code to be accepted, the automatic tests should always pass with these tools enabled (and the code rules therefore be enforced). That being said, I am open to argument and changing my stands here if you would have a strong case for not enabling it by default.

One more thing to add, mypy is being added primarily because I got fond of adding `# type: ignore` in our code every time we import this library so even if we could technically disable pycodestyle and pylint in tests, I think that we kindof need mypy to be "happy" all the time, so we can rely on the type hints of the imported code used in our apps.

Reply:
The pylist is enabled again, so we can resolve this discussion, right?

Reply:
I'd just expect those not to be enabled by default from config files.

Totally agree that all those checks should be enforced for code to be accepted. IMHO `pytest --mypy --pycodestyle --pylint` should be the test instruction in CI, but not the default for local runs.

Just a comment, no strong opinion about it!